cmake/draco_install.cmake

```diff
@@ -84,12 +83,24 @@ macro(draco_setup_install_target)
   endif()
 
   if(MSVC)
-    install(TARGETS draco DESTINATION "${libs_path}")
+    install(
+      TARGETS draco
+      RUNTIME DESTINATION "${bin_path}"
+      ARCHIVE DESTINATION "${libs_path}"
+      LIBRARY DESTINATION "${libs_path}"
+    )
   else()
-    install(TARGETS draco_static DESTINATION "${libs_path}")
+    if(NOT BUILD_SHARED_LIBS OR DRACO_INSTALL_STATIC)
```

Comment:
Why does the static library install behavior need to change now that you have the DESTINATION flags for the shared lib case above and below? I think the DRACO_INSTALL_STATIC flag can be dropped from the PR.

Reply:
It's unrelated to dll in bin dir. Installing both static & shared is a no-go in conan package manager. With this option we can avoid to patch `draco` forever.

Reply:
I think you need to automate application of that patch downstream-- conan having a weird obsession with disallowing install of static libs does not warrant maintenance and testing of a configuration.

Reply:
conan doesn't forbid installation of static, it forbids to install both static & shared in the same prefix.

Reply:
Right, and I think that's a conan problem. Draco shouldn't be responsible for maintaining and testing a configuration because of an arbitrary rule imposed by a downstream package manager.

Reply:
reverted in https://github.com/google/draco/pull/838/commits/586fa33968b6e7259ab651de4be7df46f0338bd5